packages/components/checkboxes/template.njk

```diff
@@ -12,39 +12,39 @@
    aria-describedby – for example hints or error messages -#}
 {% set describedBy = "" %}
 
-{% set isConditional = false %}
-{% for item in params.items %}
-  {% if item.conditional %}
-    {% set isConditional = true %}
-  {% endif %}
-{% endfor %}
+{#- Jinja compatability - avoid referencing params.items/params.values directly
+  as it clashes with dict.items() and dict.values() -#}
+{% set items = params["items"] if "items" in params else [] %}
+{% set values = params["values"] if "values" in params else [] %}
+
+{% set isConditional = items | selectattr("conditional") | list | length > 0 %}
```

Comment:
❗️This expression doesn't generalise to the radios component, which is called with an object like this:

```json
"items": [
  {
      "value": "email",
      "text": "Email",
      "conditional": {
      "html": emailHtml
    }
  }
]
```